lending_pool/src/lib.rs

```diff
@@ -70,13 +69,15 @@ pub trait LendingPool:
         self.require_non_zero_address(&initial_caller);
 
         let borrow_token_pool_address = self.get_pool_address(&asset_to_borrow);
-        let loan_to_value = self.get_loan_to_value_exists_and_non_zero(&collateral_token_id);
+        let loan_to_value = self.get_loan_to_value_exists_and_non_zero(&payment_lend_id);
 
         //L tokens for a specific token X are 1:1 with deposited X tokens
-        let collateral_amount = payment_amount.clone();
-        let collateral_tokens = TokenAmountPair::new(collateral_token_id, 0, collateral_amount);
         self.liquidity_pool_proxy(borrow_token_pool_address)
-            .borrow(initial_caller, collateral_tokens, loan_to_value)
+            .borrow(
+                initial_caller,
+                TokenAmountPair::new(payment_lend_id.clone(), 0, payment_amount.clone()),
```

Comment:
Since the destination already has these values as payments, I think this `TokenAmountPair` argument should be removed entirely from the liquidity pool `borrow`.

Reply:
Fixed1 :pray: